### ISSUE TITLE
[BUG] Scaled CRPS is possibly incorrect

### ISSUE THREAD

**Issue by elephaint** [Nixtla/utilsforecast#140]
[Our implementation of scaled crps](https://github.com/Nixtla/utilsforecast/blob/3b51d3a883af35ed75920dfe350f5e0b1717f6ef/utilsforecast/losses.py#L746) scales each timeseries according to the absolute value of the observed values of each timeseries, but the [original GluonTS implementation](https://github.com/awslabs/gluonts/blob/12089c7e626dd7062684bdfa82faac32da2a9ec8/src/gluonts/nursery/tsbench/src/tsbench/forecasts/metrics.py#L71) (and [the one we also follow in hierarchicalforecast](https://github.com/Nixtla/hierarchicalforecast/blob/375f7179deccebc128bc15f56513284a4df877d9/hierarchicalforecast/evaluation.py#L241)) scales the CRPS based on the norm calculated across all timeseries.

Is this intentional? 

To make things concrete, the following shows how the code for a Pandas DF should be adapted to reflect the GluonTS / HF behavior.

``` diff
loss = loss.set_index(id_col)
sizes = sizes.set_index(id_col)
assert isinstance(df, pd.DataFrame)
- norm = df[target_col].abs().groupby(df[id_col], observed=True).sum()
+ norm = df[target_col].abs().sum()
- res = 2 * loss.mul(sizes['counts'], axis=0).div(norm + eps, axis=0)
+ res = 2 * loss.mul(sizes['counts'].sum(), axis=0).div(norm + eps, axis=0)        
res.index.name = id_col
res = res.reset_index()
```

**Comment by jmoralez** [comment]
I think the whole point of having a scale is for the metric to be comparable across series, similar to MASE. If you use a global scale you're just dividing everything by a constant and the metric is dominated by series with large values.
